tools/ci_build/build.py

```diff
@@ -424,8 +424,8 @@ def convert_arg_line_to_args(self, arg_line):
     parser.add_argument(
         "--enable_address_sanitizer", action="store_true", help="Enable address sanitizer. Windows/Linux/MacOS only."
     )
-    # The following feature requires installing some special Visual Studio components that do not get installed by default. Therefore the options is default OFF.
-    parser.add_argument("--enable_qspectre", action="store_true", help="Enable Qspectre. Windows only.")
+    # The following flag is mostly designed to be used in ONNX Runtime's Azure DevOps/Github build pipelines. Its main purpose is to make the built binaries pass BinSkim scan.
+    parser.add_argument("--use_preset_compile_flags", action="store_true", help="Use preset compile flags.")
```

Comment:
maybe put the content of the comment into the help text itself? if this option is only for binskim builds (and intended to stay that way), maybe name it something more specific like `--use_binskim_compliant_compile_flags`? I think "preset" is a little vague.